<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogModifyTable1</class>
 <widget class="QDialog" name="DialogModifyTable1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>231</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>WenQuanYi Micro Hei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>修改项目</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>识别1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>59</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>说明 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem0">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditItem1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btAddConfirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定写入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btQuit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>261</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>录音更新</string>
   </property>
   <widget class="QPushButton" name="btRecordStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录音</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btRecordFinish">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>录音完成</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>更新录音</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
